src/workers/postUpdated.ts

```diff
@@ -715,7 +720,11 @@ const worker: Worker = {
           await fixData({
             logger,
             entityManager,
-            data,
+            data: {
+              ...data,
+              // pass resolved post id or fallback to original data
+              post_id: postId || data.post_id,
+            },
```

Comment:
If message without `post_id` and `source_id` arrives it would fail to resolve source because source is resolved over one of those filelds

this fixes the case where we resolve postId from yggdrasil but do not pass it in `fixData`